cerbero/build/cookbook.py

```diff
@@ -177,7 +177,8 @@ def update_step_status(self, recipe_name, step):
         @type step: str
         '''
         status = self._recipe_status(recipe_name)
-        status.steps.append(step)
+        if not step in status.steps:
+            status.steps.append(step)
```

Comment:
Just to know, does it fixes the spurious "step already done" when all steps where done?